src/git/install.sh

```diff
@@ -26,6 +26,8 @@ fi
 # Get an adjusted ID independent of distro variants
 if [ "${ID}" = "debian" ] || [ "${ID_LIKE}" = "debian" ]; then
     ADJUSTED_ID="debian"
+elif [ "${ID}" = "alpine" ]; then
+    ADJUSTED_ID="alpine"
 elif [[ "${ID}" = "rhel" || "${ID}" = "fedora" || "${ID}" = "mariner" || "${ID_LIKE}" = *"rhel"* || "${ID_LIKE}" = *"fedora"* || "${ID_LIKE}" = *"mariner"* ]]; then
```

Comment:
Can we update as well?
https://github.com/devcontainers/features/blob/4407a4d98381b74890edc3bfabcadbb48f7873c1/src/git/NOTES.md?plain=1#L5